src/backend/lima.ts

```diff
@@ -2086,12 +2099,20 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
   async stop() {
     if (this.cfg?.enabled) {
       try {
-        await this.vm.execCommand({ root: true, expectFailure: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+        const script = 'if [ -e /etc/init.d/k3s ]; then /sbin/rc-service --ifstarted k3s stop; fi';
```

Comment:
How about `'test -e /etc/init.d/k3s && /sbin/rc-service --ifstarted k3s stop` ?

Also, don't you want to do this in `LimaBackend::stop` as well?

Reply:
`foo && bar` would still fail if `foo` is falsy.  So I'm explicitly using `if; then` instead.

I'm not doing this in `LimaBackend::stop` because that didn't seem necessary (because we should have k3s installed already).

Reply:
Makes sense.